pages/models.py

```diff
@@ -2938,4 +2939,50 @@ class MathQuizPage(Page):
         APIField('search_description'),
     ]
 
-    parent_page_type = ['pages.HomePage']
\ No newline at end of file
+    parent_page_type = ['pages.HomePage']
+
+
+class LLPHPage(Page):
+    heading = models.CharField(max_length=255)
+    subheading = models.TextField()
+    signup_link_href = models.URLField()
+    signup_link_text = models.CharField(max_length=255)
+    book_cover = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+    book_heading = models.CharField(max_length=255)
+    book_description = models.TextField()
```

Comment:
Rich text for book_description to allow for paragraphing.
Also add:

- book slug, which is used to link to the book details page
- text for that link (I call them infoLinkSlug and infoLinkText)
- Two icon links, where each has a URL, an image, and alt text (these are the OpenStax Tutor and Bill of Rights Institute icons, which are links to their respective page/site)

Reply:
Done - except the two icon fields. I'm making a card to make those editable at a later time.